docs/content/docs/how-to/creating-catalogs.md

```diff
@@ -31,6 +31,8 @@ Table Store catalogs currently support two types of metastores:
 * `filesystem` metastore (default), which stores both metadata and table files in filesystems.
 * `hive` metastore, which additionally stores metadata in Hive metastore. Users can directly access the tables from Hive.
 
+See [CatalogOptions]({{< ref "docs/maintenance/configurations#catalogoptions" >}}) for detailed options when creating a catalog.
```

Comment:
remove this?

Reply:
I think this is a necessary information. This change is not related to Hive.